server/utils/helpers/validatorHelper.js

```diff
@@ -129,9 +129,54 @@ const updateUserRules = (userInput) => {
   return false;
 };
 
+const createArticleRules = (userInput) => {
+  // Validator rules
+  const rules = {
+    title: 'required|string',
+    description: 'required|string',
+    body: 'required|string',
+  };
+  // Setting up customized descriptive messages
+  const validation = new Validator(userInput, rules, {
+    'required.title': 'Please specify a title for your Article',
```

Comment:
There is a `validationErrors` object at the top of this file to help with the abstraction of validation error messages. This will help reduce repetition of code.

Reply:
Noted